### ISSUE TITLE
activityIndicatorColor shouldn't be deprecated

### ISSUE THREAD

**Issue by spybart** [jdg/MBProgressHUD#409]
I just want to express my opinion that by deprecating this property and forcing user to customize UIActivityIndicator color through UIAppearance, you are taking away flexibility from the user.

Setting color through UIAppearance basically means you are limited to the same color throughout the entire app, which may not be desirable. It is nice to be able to set the color for a specific indicator, based on the colors of the page it is being shown in.

**Comment by jjxtra** [comment]
Agreed. I still haven't figured out how to change the bezel color vs. the activity indicator color, even after reading through the documentation and deprecation warnings. Very frustrated with the new changes in 1.0.

**Comment by jjxtra** [comment]
Try setting content color property on the hud

**Comment by spybart** [comment]
@jjxtra That is not the same thing. There are definitely certain colors for the hud or background where it would be favorable to customize the activity indicator color as well.

**Comment by CaiChenghan** [comment]
@spybart 
First,you should set `UIActivityIndicatorView`'s  `indicatorColor`   like this 
```
[UIActivityIndicatorView appearanceWhenContainedInInstancesOfClasses:@[[MBProgressHUD class]]].color = [UIColor whiteColor];
```
Then,start use the `MBProgressHUD`
```
 MBProgressHUD *hud = [MBProgressHUD showHUDAddedTo:view animated:YES];
```
It's work in target Class, not all Class

**Comment by spybart** [comment]
> @spybart
> First,you should set `UIActivityIndicatorView`'s `indicatorColor` like this
> 
> ```
> [UIActivityIndicatorView appearanceWhenContainedInInstancesOfClasses:@[[MBProgressHUD class]]].color = [UIColor whiteColor];
> ```
> Then,start use the `MBProgressHUD`
> 
> ```
>  MBProgressHUD *hud = [MBProgressHUD showHUDAddedTo:view animated:YES];
> ```
> It's work in target Class, not all Class

That still isn't a solution. It could be desirable to have multiple indicator colors inside the same class.
